src/Inciter/AMR/Loggers.h

```diff
@@ -0,0 +1,19 @@
+///TODO: Add header
+#ifndef Loggers_h
+#define Loggers_h
+
+#include <iostream>
+
+#ifdef ENABLE_TRACE
+#define trace_out std::cout << "__TRACE: "
+#else
+#define trace_out while(0) std::cout
+#endif /* ENABLE_TRACE */
+
+#ifdef ENABLE_DEBUG
+#define debug_out std::cout << "__DEBUG: "
+#else
+#define debug_out while(0) std::cout
+#endif /* ENABLE_DEBUG */
+
+#endif
```

Comment:
I don't think we should perform anything major as part of this PR, just wanted to let folks now that I have also thought about adding some kind of logging facility, see https://github.com/quinoacomputing/quinoa/issues/3.

Should we care about logging as much as pulling in a TPL for it do it well? Or just pull our own but useable from the whole code and not only from AMR? Any thoughts? (Perhaps discuss in #3).